o2a/mappers/start_mapper.py

```diff
@@ -13,16 +13,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Maps Oozie start node to Airflow's DAG"""
-from typing import Set
+from typing import Set, Tuple, List
+from xml.etree.ElementTree import Element
 
+from airflow.utils.trigger_rule import TriggerRule
+
+from o2a.converter.relation import Relation
+from o2a.converter.task import Task
 from o2a.mappers.base_mapper import BaseMapper
 
 
 class StartMapper(BaseMapper):
+    """Maps start node"""
+
+    def __init__(self, oozie_node: Element, name: str, dag_name: str):
```

Comment:
What is purpose of this code? I think, It useless, because BaseMapper provide a good constructor.

Reply:
It was to have dummy trigger_rule. But after your comment I found that it will be even easier to simplify the whole mapper creation instead and hard-code the rule when the Start Mapper is created int that one place it was used. On the way I also removed CONTROL_MAP map for the control mappers as it turned out to be completely not needed with this change. See abb241def467a5cbef041ebe6c78d33a7c97541e

Reply:
Please @mik-laj resolve this is if you think the discussion is resolved.